lib/pure/ptrops.nim

```diff
@@ -0,0 +1,32 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2019 Nim contributors
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## This module offers operators for performing arithmetic on raw pointer types
+## in Nim. Nim is *not* C: using pointer arithmetic in Nim should be avoided (if
+## possible, use `array[T]`, `ptr UncheckedArray[T]`, etc.), however, pointer
+## arithmetic is sometimes desirable for low-level ops/FFI with C. **Pointer
+## arithmetic is a very sharp tool. This module is unsafe, comes without
```

Comment:
```suggestion
##
## Risks
## -----
##
## - Pointer Arithmetic is a very sharp tool. This module is unsafe.
## - Nim is *not* C: Using Pointer Arithmetic in Nim should be avoided.
## - Other reason why it can be problematic.
## - Other reason why it can be problematic.
```